examples/main.tf

```diff
@@ -189,6 +189,40 @@ resource "instaclustr_cluster" "example_kafka" {
   wait_for_state = "RUNNING" // the required state of the cluster before doing the bundle user password updates
 }
 
+// Use Kafka with Karapace Schema Registry
+resource "instaclustr_cluster" "example_kafka_with_karapace_schema_registry" {
+  cluster_name = "test_kafka_with_karapace"
+  node_size = "KFK-PRD-r6g.large-250"
+  data_centre = "US_WEST_2"
+  sla_tier = "NON_PRODUCTION"
```

Comment:
**[QUESTION]** Isn't `KFK-PRD-r6g.large-250` production node size?

Reply:
Yeah, it's just an example, I don't think it does matter. We also have production node sizes in other examples.

Reply:
i prefer consistency but okays.